templates/label-landscape.php

```diff
@@ -0,0 +1,262 @@
+<?php
```

Comment:
🤔 Do we really need a completely separate template?  How much code, especially HTML and CSS, are we duplicating?  Can't we include shared styles from a separate file (but still output them inline)?

@juanlopez4691 Please let me know if it doesn't make sense to do this (DRY) for this plugin.

Reply:
@sun When I look at a comparison between `label-portrait.php` and `label-landscape.php`, there are three differences in the HTML – all other differences are in CSS.

What do you think, @juanlopez4691?

Reply:
@moritzbappert If the HTML differences are so few, then a single template and some conditionals (based on an `$orientation` variable?) to select the relevant markup for each case should be enough. The idea by @sun about conditionally including separate files to inject inline CSS sounds right to me, too.

Reply:
Hey @sun & @juanlopez4691, I've just completed your suggested changes with the latest commit. ;)